### ISSUE TITLE
ToTK: Infinite Stamina flag missing from Horse Editor

### ISSUE THREAD

**Issue by ApacheThunder** [marcrobledo/savegame-editors#252]
There's a few horses that have this. The Gannondorf horse, the other new big horse in ToTK and of coarse Satori (at least I believe so it's actually been a while since I went and rode him).

Anyways it appears the flag for setting the stamina to the infinite type that the big horses is missing. But the editor displays this when I open a save that has one of these horse:

![image](https://github.com/marcrobledo/savegame-editors/assets/11767416/34d6f58c-0e08-4a86-af86-69f58fdb2b1d)

Note the [0] in the Stamina Drop down. It only appears for him and I can't set it to that for other horses so it's a flag/setting the editor isn't configured for currently.

I guess put it on the todo list. :P

EDIT: Oh now that I recall Satari did have a stamina bar but it refills instantly so it's actually a unique flag from all the other infinite stamina horses.

Tested setting all the equipment to none for hte stalhorse and Satori slots and the game bugs out and shows me a camera view somewhere off in narnia underground and the horse never really appears. (at least the game doesn't crash. :P )

So yeah I bet they might work in BoTW's stables if the equipment was set on them. I may fire up BoTW at some point and play around with the BoTW editor to see if they actually work as is with proper equipment set. :D

**Comment by ApacheThunder** [comment]
For clearification, this is what occurs if you set any of the equipment flags to none:

![image](https://github.com/marcrobledo/savegame-editors/assets/11767416/9471bdb4-9bed-4844-9feb-dfcd8ae10b3d)

Really odd. And I see glitched selections in the equipment menu for 9,999 rupees. I might try and hack in rupees to see what happens but I would recommend adding asterisk next to the none flags (or end users from setting them) as they don't really work.

**Comment by marcrobledo** [comment]
> There's a few horses that have this. The Gannondorf horse, the other new big horse in ToTK and of coarse Satori (at least I believe so it's actually been a while since I went and rode him).
Added infinite stamina to the editor, thank you! Though I'm not sure it will work for normal horses.

> Tested setting all the equipment to none for hte stalhorse and Satori slots and the game bugs out and shows me a camera view somewhere off in narnia underground and the horse never really appears. (at least the game doesn't crash. :P )
Haha!

> So yeah I bet they might work in BoTW's stables if the equipment was set on them. I may fire up BoTW at some point and play around with the BoTW editor to see if they actually work as is with proper equipment set. :D
I haven't checked but I'm pretty sure BOTW crashed because of the icons. If I recall correctly, BOTW had hardcoded icons for every horse. TOTK now generates the icon dynamically when the horse is registered. That's probably why now you can edit unusable horses in the editor: because it loads a normal horse icon. I can't confirm though.




> For clearification, this is what occurs if you set any of the equipment flags to none:
I've removed the saddle and bridle None equipment. Does that glitch happen when mane is set to none? It would be funny to see if a normal horse can load without a mane.

**Comment by ApacheThunder** [comment]
I believe I tried every combo. If ANY of the equipment slots are set ot none, it breaks and the horse can't be used without that bug happening. :(

Hopefully the flags for bears, and stuff will be found. Would love to try putting one of those in a stable. :P

**Comment by ApacheThunder** [comment]
Ok the new version of the editor can set the Mane to none and this doesn't break the horses but at the same time it doesn't actually remove the mane. Maybe the non flags weren't correct or something. Here's what they show up as in the new editor:

![image](https://github.com/marcrobledo/savegame-editors/assets/11767416/5213b8f4-5c08-42af-9e4d-6251b52db869)


The inifinte stamina thing seems to work on every horse I put it on though they boot me off if I try to make them go faster. Original White Stalion doesn't do that. Just makes angry noises when you try and make him go any faster. :P

Maybe it's just horse temperament which doesn't appear to be configurable in the editor yet.

Satori's infinite stamina must be a seperate flag because I do recall now that his stamina bar was special in that he had one that would refill instantly so unlike normal inifinite staminia horses like the giant horses he can go in that final fast speed that using the staminia bar usually does while the infinite stamina horses can't go to the full galop speed.

I guess someone will have to find the flag for that one.

**Comment by ApacheThunder** [comment]
Ok I see what is happening. Stalhorse gained a mane the moment I went into the equipment menu. But then found out why Satori was still getting a mane even though I set it to none. I switched the stalhorse into the slot Satori used to be in and put Satori in the stalhorse slot that I saw had a working "none" mane setting. So now I got a Satori without a double glitched mane...as long as I don't visit the customize horse menu. :P


Oh also saw Donkey in the drop menu with the need test in all caps so gave him a go:

![image](https://github.com/marcrobledo/savegame-editors/assets/11767416/e453578f-17ad-4f6b-bfa4-08af1d2e47c8)


Sadly he does not work. :(
